code/js/i18n/en.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
Files en.json and cs.json must be stored in folder i18n ?
If not rename it to localization or translation.

Reply:
folder i18n has been renamed to localization